en/docs/install-and-setup/setup/user-stores/setting-up-a-userstore-in-icp.md

```diff
@@ -0,0 +1,306 @@
+type = "read_only_ldap"
+read_groups = "true"
+```
+
+The parameters used above are explained below.
+
+<table>
+	<tr>
+		<th>Parameter</th>
+		<th>Value</th>
+	</tr>
+	<tr>
+		<td>
+			<code>connection_url</code>
+		</td>
+		<td>
+			The URL for connecting to the LDAP. If you are connecting over LDAPS (secured LDAP), you need to import the certificate of the user store to the truststore (<code>client-truststore.jks</code> by default). See the instructions on how to <a href="{{base_path}}/install-and-setup/setup/security/importing-ssl-certificate/#importing-ssl-certificates-to-a-truststore">add certificates to the truststore</a>.
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<code>connection_name</code>
+		</td>
+		<td>
+			The username used to connect to the user store and perform various operations. This user needs to be an administrator in the user store. That is, the user requires write permission to manage add, modify users and perform search operations on the user store. The value you specify is used as the DN (Distinguish Name) attribute of the user who has sufficient permissions to perform operations on users and roles in LDAP.
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<code>connection_password</code>
+		</td>
+		<td>
+			Password for the connection username.
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<code>user_search_base</code>
+		</td>
+		<td>
+			The DN of the context or object under which the user entries are stored in the user store. When the user store searches for users, it will start from this location of the directory.
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<code>type</code>
+		</td>
+		<td>
+			Use one of the following values. </br></br>
+			<code>read_only_ldap</code>: The LDAP connection does not provide write access.</br>
+			<code>read_write_ldap</code>: The LDAP connection provides write access.
+		</td>
+	</tr>
+</table>
+
+See the [complete list of parameters]({{base_path}}/reference/config-catalog-mi/#external-user-store) you can configure for the LDAP user store.
+
+## Configure an RDBMS user store
+
+!!! note "Before you begin"
+    Disable the file-based user store as shown below using the `<ICP_HOME>/conf/deployment.toml` file.
+
+       ```toml
+       [internal_apis.file_user_store]
+       enable = false
+       ```
+
+If you are already using a JDBC user store (database) with another WSO2 product ([WSO2 API Manager](https://wso2.com/api-management/), [WSO2 Identity Server](https://wso2.com/identity-and-access-management/), or an instance of [WSO2 Enterprise Integrator 6.x.x](https://wso2docs.atlassian.net/wiki/spaces/EI660/overview)), you can connect the same database to the ICP. Alternatively, you can create a new RDBMS user store and connect it to the ICP.
+
+1. To set up a new RDBMS, select the preferred RDBMS type and follow the instructions.
+
+	!!! Tip
+		If you already have an RDBMS user store set up, you can skip this step.
+
+	- [Setting up a MySQL database]({{base_path}}/install-and-setup/setup/databases/setting-up-mysql)
```

Comment:
Should we unify these pages to mention both MI an ICP?

Reply:
Updated the page structure via https://github.com/wso2/docs-mi/pull/962/commits/46fb5a3a15382eccf1551c3250b7c4653feb6f54.
<img width="247" alt="updated-page-structure" src="https://github.com/user-attachments/assets/32578a40-1e68-49db-8ae3-0fc4a66fede9">